lib/diagnostic.js

```diff
@@ -47,7 +48,8 @@ let scopes = [/* your response here */];
 // executing it.
 //
 let fireOnAlderaan = function () {
-  // your code here
+  $ ("button").on('click', fire.bind(fire))
```

Comment:
This is the right code to have the set a click handler on a button, but again, it's not what the question was asking for. Please read directions carefully.

The following calls the `fire` method with `alderaan` as the context.
```js
let fireOnAlderaan = function () {
  return deathStar.fire.call(alderaan);
};
```